### ISSUE TITLE
cannot open a just saved txt file in this version on Windows 10

### ISSUE THREAD

**Issue by cgeugten** [notepad-plus-plus/notepad-plus-plus#5430]
### Description of the Issue
I create a text file and I save it.
Now I try to open the saved file and NPP fails.

I close NPP.
when I reopen NPP tries to open the files that were open when it was closed.
it fails.

Note:
I had a similar problem with TXT files in another application.
This might be just a local problem, of my own desktop.
This is what I do not yet know.

### Steps to Reproduce the Issue
see desription and screendump

1. 
2. 
3. 

### Expected Behavior
normal behaviour is that the files can be read.

### Debug Info Info rmation
Notepad++ v7.6.4   (64-bit)
Build time : Mar  6 2019 - 02:58:24
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)

Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll 

![Error_msgbox](https://user-images.githubusercontent.com/23422238/54477844-fe4fe600-480b-11e9-930c-d5d7d00e295a.PNG)

**Comment by MetaChuh** [comment]
welcome to the notepad++ issue tracker, @cgeugten 

i can not reproduce this with notepad++ 7.6.4. 64bit on windows 10.

**please test the following, to verify, if a clean notepad++ would work on your system.**

* download the notepad++ 7.6.6. 64 bit portable version from [>>> here <<<](https://notepad-plus-plus.org/repository/7.x/7.6.6/npp.7.6.6.bin.x64.zip).
(the portable version does not require any installation and runs completely independent and isolated from your installed version. all custom settings, as well as any stuff you might want to try at the portable version, will not modify your installed version)

* extract `npp.7.6.6.bin.x64.zip` to your desktop.

* **important note:** make sure to close all instances of notepad++ that might be running, before starting the portable version at the next step, to be sure you are using the portable version for this test.

* open the extracted `npp.7.6.6.bin.x64` folder and start this portable version by double-clicking on `notepad++.exe` inside this folder.
(note: notepad++.exe will just be seen as notepad++, if you have enabled to hide all known file extensions at your explorer settings)

if this portable version works, if you try to save a file to your desktop and reopen it: please post your pc problem at the notepad++ community forum (link at the bottom), as it is not code related.

-----
```
general notes:

please close your issues as soon as you are satisfied,
to help keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if a related topics exists. you are welcome to post at either an eventually similar
topic, or create a new topic at Help Wanted or General Discussion.
```
[<kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd>](https://notepad-plus-plus.org/community/)
